packages/@glimmer/opcode-compiler/lib/opcode-builder/helpers/shared.ts

```diff
@@ -39,7 +39,7 @@ export function CompileArgs(
 
   if (atNames) flags |= 0b1000;
 
-  if (blocks) {
+  if (blocks.hasAny) {
```

Comment:
I think this meant that all component invocations were tagged as having args. I'm not sure how much
that matters, but if we're going to do a check, we may as well do it properly.